app/packages/core/src/components/Actions/Common.tsx

```diff
@@ -35,6 +35,7 @@ export const ActionOption = React.memo(
     return (
       <ItemAction
         title={title ? title : text}
+        as={href ? "a" : undefined}
```

Comment:
I'm not sure why I added an `href` prop to `ItemAction`, but it can probably  be removed. It's not used anywhere and `href` links aren't safe in the App because we distribute an Electron desktop app version and `a` links don't work there.

External links can be created with this [component](https://github.com/voxel51/fiftyone/blob/develop/app/packages/components/src/components/ExternalLink/ExternalLink.tsx).